style.css

```diff
@@ -188,3 +201,8 @@ header {
 .legend-bobs {
   background: linear-gradient(to right bottom, #87d059 50%, #b8dfa0 50%);
 }
+
+html.transiton, html.transition *, html.transition *:before, html.transition *:after{
+    transition: all 300ms !important;
+    transition-delay: 0 !important;
+}
```

Comment:
I would avoid using !important as much as possible. Nothing is being overridden here so it shouldn't be necessary.